precompile/modules/initia_stdlib/sources/dex.move

```diff
@@ -214,7 +228,14 @@ module initia_std::dex {
     /// Weights sum must be 1.0
     const EINVALID_WEIGHTS: u64 = 21;
 
+    /// Recursive flash swap is not allowed
+    const ERECURSIVE_FLASH_SWAP: u64 = 22;
```

Comment:
may i ask how does recursive flash swap work?

Reply:
We store FlashSwap object under pair address when flash swap executed and check this is exist or not at flash_swap call.